buildspec_publish_ssm.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.2         
+phases:
+  install:
+    runtime-versions:
+      golang: 1.12
+  pre_build:
+    commands:
+      - echo Publish SSM parameters
+  build:
+    commands:
+      - './scripts/publish_test.sh cicd-publish-ssm ${AWS_REGION}'
+
+      # Assume role to verify, get the credentials, and set them as environment variables
```

Comment:
Nit: this comment explains the code, but the code is pretty obvious and readable. Instead, I'd prefer a comment explaining why the code exists, like "Verify using a role in a different account, parameters should be public"